meta-oe/recipes-support/libcanberra/files/0001-Determine-audio-buffer-size-for-a-time-of-500ms.patch

```diff
@@ -0,0 +1,40 @@
+From 86488a7fc209ac08dd92c9d50a77e3330e7aedd9 Mon Sep 17 00:00:00 2001
+From: Patrick Zacharias <1475802+Fighter19@users.noreply.github.com>
+Date: Thu, 7 Nov 2024 14:03:29 +0100
+Subject: [PATCH] Determine audio buffer size for a time of 500ms
+
+On some hardware like the SGTL5000, not specifying a buffer size
+results to EINVAL being returned.
+
+This code sets the buffer time to 500ms and the period time to a fourth of that,
+or whatever is nearest to that.
+---
```

Comment:
Please add `Upstream-Status` field to patch. This is a required field these days.

```
stdio: ERROR: libcanberra-0.30-r0 do_patch: QA Issue: Missing Upstream-Status in patch
stdio: ERROR: libcanberra-0.30-r0 do_patch: Fatal QA errors were found, failing task.

```

Reply:
Would Upstream-Status: Pending be appropriate?
The mailing list archive is defunct and no updates to the project have been made on Upstream in 12 years.